<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>248</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>225</width>
    <height>248</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="minimumSize">
         <size>
          <width>191</width>
          <height>41</height>
         </size>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>8</number>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_9">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_7">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_8">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_6">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButton_14">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nimbus Sans L</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_11">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nimbus Sans L</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_12">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nimbus Sans L</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_13">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nimbus Sans L</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_10">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_15">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nimbus Sans L</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="pushButton_16">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Nimbus Sans L</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
